src/D2L.Bmx/Program.cs

```diff
@@ -282,3 +304,8 @@
 	} )
 	.Build()
 	.InvokeAsync( args );
+
+internal static partial class NativeMethods {
```

Comment:
Top Level Statement L here?

Reply:
I'd rather we wrap this along with the IsWindows, IsLinux, IsMacOS logic all in a separate file & class.

Reply:
There's a lot happening in Program.cs already

Reply:
Sure